.gitlab-ci.yml

```diff
@@ -0,0 +1,22 @@
+image: node:latest
+
+stages:
+  - build
+  - test
+
+cache:
```

Comment:
It might be beneficial to combine `install_dependencies` and `run_tests` in the same stage, transferring artifacts from one stage to another adds delay, so only is beneficial if we can parallelize afterwise.